### ISSUE TITLE
New Scenes Open in Libraries Folder Rather than "Proper" Location

### ISSUE THREAD

**Issue by john01dav** [jMonkeyEngine/sdk#62]
https://www.youtube.com/watch?v=zJ4Z4TXAJjg

This also happens with other kinds of new files (ie. filters). It doesn't happen with Java classes.

**Comment by MeFisto94** [comment]
Yeah, this issue is there for quite a while now and it's annoying.
It's part of #7 already.

Unfortunately we have no clue on where that comes from, it suddenly appeared and no one knows.
In fact this is since Assets actually _are_ a dependency and the Assets Folder is actually something custom made by us which simply displays the assets/ Directory.

So if anyone who is reading this is a bit bored, feel free to go back in the history of the repo, maybe you find the commit which broke things (or at least roughly), so we have a clue on what to look for.

**Comment by grizeldi** [comment]
Can we manually select the correct file under assets? That might be ok for a temporary fix.

**Comment by MeFisto94** [comment]
I have no clue on how to do this but yeah, maybe this could work. Maybe in conjunction with a 250ms timer so it first selects the library.

**Comment by grizeldi** [comment]
Duplicate of #98.

**Comment by MeFisto94** [comment]
See #98 for more in-detail reviews that we did

**Comment by grizeldi** [comment]
@MeFisto94 Any chances this is fixed with NB11?
